src/core.js

```diff
@@ -1,3 +1,12 @@
+// JQuery trim is deprecated, polyfill native js method if not defined
+if ( !String.prototype.trim ) {
```

Comment:
We should not pollute the prototype of builtin objects. Please instead create a local trim-function instead

Reply:
Fixed.